### PR TITLE
chore: update google-cloud-cpp-common to v0.22.1

### LINKED ISSUE #3437: [Flake] Build fails with "random_device: rdseed failed"
I was wondering if we were eating too many bits of entropy when initialization our PRNGs, I think I have the answer:

```
Running embedded Cloud Bigtable server at localhost:34903
# Populating table populate-m8nLrFe6 ............................unknown file: Failure
C++ exception with description "random_device: rdseed failed" thrown in the test body.
```

https://source.cloud.google.com/results/invocations/c874a6d0-8180-4bfd-af39-4da9801b365d

### PR DESCRIPTION
**Pull request by coryan**
This updates the version of `g-c-cpp-common` used for the CI builds, for
Bazel, and for the CMake super build. It also updates the README and
INSTALL files (which required some fixes to the markdown generator).

Fixes #3437 

---
This change is [Reviewable](https://reviewable.io/reviews/googleapis/google-cloud-cpp/3459)

### CODE REVIEW

File: google/cloud/bigtable/benchmarks/setup.cc [hunk 1]
```diff
@@ -17,19 +17,20 @@
...
 #include <iostream>
 #include <sstream>
 
 /// Supporting types and functions to implement `BenchmarkSetup`
 namespace {
 std::string FormattedStartTime() {
-  auto start = std::chrono::system_clock::now();
-  std::time_t start_c = std::chrono::system_clock::to_time_t(start);
+  auto const start = std::chrono::system_clock::now();
+  auto const start_as_time_t = std::chrono::system_clock::to_time_t(start);
```
Comment (devbww) [inline google/cloud/bigtable/benchmarks/setup.cc]: Are these edits related to the common version jump?

Comment (coryan) [reply google/cloud/bigtable/benchmarks/setup.cc]: Not really, let me revert them.
